README.md

```diff
@@ -0,0 +1,8 @@
+# Brainfuck Interpreter
```

Comment:
you can also add an example how to get your library, how to run the tests

Reply:
Done